src/plugins/legacy/polygonRoi/toolboxes/polygonRoiToolBox.cpp

```diff
@@ -597,11 +608,11 @@ void polygonRoiToolBox::showHelp() const
     
     QString main = QString("<h3>Main features</h3>")
         + QString("<ul>")
-        + QString("<li><b>Draw a contour</b>: shift+click on the data to create a contour</li>")
+        + QString("<li><b>Draw a contour</b>: click on the data to create a contour</li>")
         + QString("<li><b>Add a new label</b>: click on '+' button in the label list</li>")
         + QString("<li><b>Remove a label</b>: click on '-' button in the label list</li>")
-        + QString("<li><b>Use a new label</b>: select a label in the list, then shift+click on the data</li>")
-        + QString("<li><b>Alt+click</b>: draw a landmark at that position in all views</li>")
+        + QString("<li><b>Use a new label</b>: select a label in the list, then click on the data</li>")
+        + QString("<li><b>Alt+click</b>: draw a landmark at that position in all slices</li>")
```

Comment:
The term 'all slices' would refer, literally, to all the slices of the volume, which is not the intended meaning. The word 'views' was more appropriate, although there really is no reason to specify where the landmark will be placed. I think the user can understand that the landmark will be placed on the planes corresponding to each of the current views.

Reply:
there is only one view, this landmark is displayed indeed in all slices on a volume in one view.
The multi views in PolygonRoi has been partially added then removed, now it's like before, there is only one view.

Reply:
Ok (I was indeed visualizing the way landmarks work on vessel segmentation for example).